cmd/rekor-server/app/root.go

```diff
@@ -88,6 +94,21 @@ func init() {
 	if GitVersion != "devel" {
 		return
 	}
+	log.Logger.Debugf("pprof enabled %v", enablePprof)
+	// Enable pprof
+	if enablePprof {
```

Comment:
do we need to check if the user has requested port 6060 for the main rekor endpoint and give a helpful error message?

Reply:
For now, I would pass on this because this flag has to be turned on and the default rekor endpoint port has to be set to `6060`. The odds of that happening are less IMO. Also, we have included the docs in the command line it could help someone debug. 

Thoughts?